nntrainer/tensor/tensor.h

```diff
@@ -1588,6 +1588,20 @@ class Tensor {
     dim.setFormat(fm);
   }
 
+  /**
+   * @brief     set the memory format
+   * @param     fm format of Tensor
+   */
+  void convertDataType(TensorDim::DataType type) {
```

Comment:
Where is this function used? (cannot find it)
We have a similar function `setDataType()` but will be deprecated. (please check the details in #2498)
If this functionality is essential, we have to consider another way to enable this after refactorization. (e.g., delete FloatTensor and recreate HalfTensor)

Reply:
Thanks. it was test code.
It removed now.